### ISSUE TITLE
issue reporter wont stop opening

### ISSUE THREAD

**Issue by JulienReveillon** [spyder-ide/spyder#17454]
### What steps will reproduce the problem?

I do not want to report an issue I don't understand

### Traceback
```python-traceback
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/dist-packages/IPython/core/history.py", line 77, in catch_corrupt_db
    return f(self, *a, **kw)
  File "/usr/local/lib/python3.8/dist-packages/IPython/core/history.py", line 246, in init_db
    self.db.execute(
sqlite3.OperationalError: disk I/O error

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/local/lib/python3.8/dist-packages/spyder/plugins/ipythonconsole/widgets/main_widget.py", line 1479, in create_new_client
    client = ClientWidget(self, id_=client_id,
  File "/usr/local/lib/python3.8/dist-packages/spyder/plugins/ipythonconsole/widgets/client.py", line 147, in __init__
    self.shellwidget = ShellWidget(
  File "/usr/local/lib/python3.8/dist-packages/spyder/plugins/ipythonconsole/widgets/shell.py", line 111, in __init__
    super(ShellWidget, self).__init__(*args, **kw)
  File "/usr/local/lib/python3.8/dist-packages/spyder/plugins/ipythonconsole/widgets/debugging.py", line 195, in __init__
    super(DebuggingWidget, self).__init__(*args, **kwargs)
  File "/usr/local/lib/python3.8/dist-packages/spyder/plugins/ipythonconsole/widgets/debugging.py", line 68, in __init__
    self._pdb_history_file = PdbHistory()
  File "/usr/local/lib/python3.8/dist-packages/IPython/core/history.py", line 539, in __init__
    super(HistoryManager, self).__init__(shell=shell, config=config,
  File "/usr/local/lib/python3.8/dist-packages/IPython/core/history.py", line 219, in __init__
    self.init_db()
  File "/usr/local/lib/python3.8/dist-packages/decorator.py", line 232, in fun
    return caller(func, *(extras + args), **kw)
  File "/usr/local/lib/python3.8/dist-packages/IPython/core/history.py", line 85, in catch_corrupt_db
    elif self.hist_file.is_file():
AttributeError: 'str' object has no attribute 'is_file'
```

## Versions

* Spyder version: 5.2.2 
* Python version: 3.8.10
* Qt version: 5.15.2
* PyQt5 version: 5.15.6
* Operating System: Linux 5.13.0-30-generic

### Dependencies

```
# Mandatory:
atomicwrites >=1.2.0          :  1.4.0 (OK)
chardet >=2.0.0               :  3.0.4 (OK)
cloudpickle >=0.5.0           :  1.3.0 (OK)
cookiecutter >=1.6.0          :  1.7.3 (OK)
diff_match_patch >=20181111   :  20200713 (OK)
intervaltree >=3.0.2          :  3.1.0 (OK)
IPython >=7.6.0;<8.0.0        :  8.1.1 (NOK)
jedi >=0.17.2;<0.19.0         :  0.18.1 (OK)
jellyfish >=0.7               :  0.8.9 (OK)
jsonschema >=3.2.0            :  4.0.0 (OK)
keyring >=17.0.0              :  18.0.1 (OK)
nbconvert >=4.0               :  6.4.2 (OK)
numpydoc >=0.6.0              :  1.2 (OK)
parso >=0.7.0;<0.9.0          :  0.8.3 (OK)
pexpect >=4.4.0               :  4.8.0 (OK)
pickleshare >=0.4             :  0.7.5 (OK)
psutil >=5.3                  :  5.5.1 (OK)
pygments >=2.0                :  2.11.2 (OK)
pylint >=2.5.0                :  2.9.6 (OK)
pyls_spyder >=0.4.0           :  0.4.0 (OK)
pylsp >=1.3.2;<1.4.0          :  1.3.3 (OK)
pylsp_black >=1.0.0           :  1.0.0 (OK)
qdarkstyle =3.0.2             :  3.0.2 (OK)
qstylizer >=0.1.10            :  0.2.1 (OK)
qtawesome >=1.0.2             :  1.0.3 (OK)
qtconsole >=5.2.1;<5.3.0      :  5.2.2 (OK)
qtpy >=1.5.0                  :  2.0.1 (OK)
rtree >=0.9.7                 :  0.9.7 (OK)
setuptools >=49.6.0           :  60.9.3 (OK)
sphinx >=0.6.6                :  1.8.5 (OK)
spyder_kernels >=2.2.1;<2.3.0 :  2.2.1 (OK)
textdistance >=4.2.0          :  4.2.1 (OK)
three_merge >=0.1.1           :  0.1.1 (OK)
watchdog >=0.10.3             :  2.1.5 (OK)
xdg >=0.26                    :  0.26 (OK)
zmq >=17                      :  22.3.0 (OK)

# Optional:
cython >=0.21                 :  0.29.14 (OK)
matplotlib >=2.0.0            :  3.5.1 (OK)
numpy >=1.7                   :  1.21.2 (OK)
pandas >=1.1.1                :  1.3.3 (OK)
scipy >=0.17.0                :  1.7.1 (OK)
sympy >=0.7.3                 :  1.8 (OK)
```

**Comment by ccordoba12** [comment]
Hey @JulienReveillon, thanks for reporting. You have an incompatible version of IPython, as can be seen in your list above:

    IPython >=7.6.0;<8.0.0        :  8.1.1 (NOK)

And the error you reported above is related precisely to IPython.

You're probably getting a dialog at every startup saying that that version is incorrect, that you need to fix it and that errors that come from it should not be reported here.

Anyway, please downgrade to the latest IPython 7 version available (7.32.0, if I'm not mistaken) and try again.

**Comment by ccordoba12** [comment]
Closing due to lack of response.
